docs/RFCS/00000000_template.md

```diff
@@ -51,6 +52,11 @@ This is the technical portion of the RFC. Explain the design in sufficient detai
 (You may replace the section title if the intent stays clear.)
 
 - Its interaction with other features is clear.
+- It covers where this feature may be surfaced in other areas of the product
+   - What metrics should be measured? Should they be surfaced in the admin UI?
+   - How will we debug issues? Do these metrics need to be surfaced anywhere?
```

Comment:
What do we mean by "metrics" here and are you distinguishing this from telemetry?

Reply:
Nate: The word "metric" is used throughout the project (on the engineering side) with an unambiguous meaning, and yes this is separate from telemetry.

FYI / reminder: the "metrics" are samples of instantaneous measurements in a CockroachDB cluster, collected over time and stored in our internal time series database. Telemetry usually performs aggregate queries over the stored metrics and reports a summary to our reg server.

Reply:
Diana: do not phrase what you want as questions in here. So instead of:

"What metrics should be measured? Should they be surfaced in the admin UI?"

Write:

"Your RFC must consider and propose a set of metrics to be collected, if applicable, and suggest which metrics would be useful to users and need to be exposed in a public interface."

(Also avoid using the phrase "the admin UI" when you really mean "a public interface". The Admin UI is a specific sub-product and CockroachDB provides many more ways to expose data to users than just the admin UI.)

Reply:
ditto for the other thing about debugging:

"Your RFC should outline how you propose to investigate when users run into related issues in production. If you propose new data structures, suggest how they should be checked for consistency. If you propose new asynchronous subsystems, suggest how a user can observe their state via tracing. In general, think about how your coworkers and users will gain access to the internals of the change after it has happened to either gain understanding during execution or troubleshoot problems."